src/components/Lesson/Lesson.js

```diff
@@ -145,7 +167,9 @@ const mapStateToProps = (state, {path, params}) => ({
   authorName: getAuthorName(state, params),
   translatorName: getTranslatorName(state, params),
   isReadme: state.context.readmeContext.keys().indexOf('./' + path + '.md') !== -1,
-  isStudentMode: state.isStudentMode
+  isStudentMode: state.isStudentMode,
+  checkedCheckboxes: getNumberOfCheckedCheckboxes(state, createCheckboxesKey(path)),
+  totalCheckboxes: getTotalNumberOfCheckboxes(state, createCheckboxesKey(path)),
```

Comment:
Jeg fant en liten bug i `getTotalNumberOfCheckboxes`. Jeg er usikker på grunnen, men kanskje oppgave-siden har endret seg? For jeg fikk "31/56" når jeg hadde checked alle boksene. Jeg måtte slette localstorage, og DA ble det riktig igjen. Kanskje vi burde scannet alle checkboxes eller noe ved oppstart, eller når man checker en ny box? Uansett, ikke denne PR'en, må bli et eget issue.

Reply:
Ja jeg merket samme buggen som gjorde at jeg måtte slette localstorage. Lager et issue på det. Dette problemet er nok bare hvis oppgaven har blitt endret